Source/Scene/Expression.js

```diff
@@ -928,6 +1029,10 @@ define([
         var right = this._right.evaluate(frameState, feature);
         if ((right instanceof Color) && (left instanceof Color)) {
             return Color.mod(left, right, ScratchStorage.getColor());
+        } else if ((right instanceof Cartesian4) && (left instanceof Cartesian4)) {
+            // TODO : make mod a built-in Cartesian function?
```

Comment:
Up to you, but I'm not sure it is worth it until/less we have another use case since we would also have to add it to `Cartesian2`, `Cartesian3`, etc.